PS 2 Elshana

```diff
@@ -0,0 +1,183 @@
+# relationship: Wife, Own-child, Husband, Not-in-family, Other-relative, Unmarried.
+# race: White, Asian-Pac-Islander, Amer-Indian-Eskimo, Other, Black.
+# sex: Female, Male.
+# capital-gain: continuous.
+# capital-loss: continuous.
+# hours-per-week: continuous.
+# native-country: United-States, Cambodia, England, Puerto-Rico, Canada, Germany, Outlying-US(Guam-USVI-etc), India, Japan, Greece, South, China, Cuba, Iran, Honduras, Philippines, Italy, Poland, Jamaica, Vietnam, Mexico, Portugal, Ireland, France, Dominican-Republic, Laos, Ecuador, Taiwan, Haiti, Columbia, Hungary, Guatemala, Nicaragua, Scotland, Thailand, Yugoslavia, El-Salvador, Trinadad&Tobago, Peru, Hong, Holand-Netherlands.
+# salary: >50K,<=50K
+
+# Further description is available in adult.names file
+
+# 1) read the data, make sure columns are read in the right format, i.e. character, double, etc.
+library(readr)
+adult <- read_csv("C:/Users/user/Documents/GitHub/RfEX_stud_version/ps2/adult.data", col_names = FALSE)
+View(adult)
+library(magrittr)
+
+library(tidyverse)
+
+names(adult)[4]<-paste("Education")
+names(adult)[5]<-paste("educ.numbers")
+names(adult)[9]<-paste("race")
+names(adult)[10]<-paste("sex")
+names(adult)[1]<-paste("Age")
+names(adult)[12]<-paste("0")
+names(adult)[8]<-paste("family.condition")
+names(adult)[15]<-paste("salary")
+names(adult)[2]<-paste("workclass")
+names(adult)[6]<-paste("maritalstatus")
+names(adult)[7]<-paste("occupation")
+names(adult)[8]<-paste("relationship")
+names(adult)[11]<-paste("capitalgain")
+names(adult)[12]<-paste("capitalloss")
+names(adult)[13]<-paste("hoursperweek")
+names(adult)[14]<-paste("nativecontry")
+names(adult)[3]<-paste("fnlwgt")
+attach(adult)
+
+# 2) How many men and women are represented in the dataset? 
+
+a <- sum(adult$sex == "Female")
+a
+b <- sum(adult$sex == "Male")
+b
+# 3) What is the average age of women
+
+mean(Age[sex == "Female"])
+
+# 4) What are the mean and standard deviation of age for those who earn more than 50K per year
+# and those who earn less than 50K per year
+mean(Age[salary == ">50K"])
+sd(Age[salary == ">50K"])
+mean(Age[salary == "<=50K"])
+sd(Age[salary == "<=50K"])
+# 5) Is it true that people who earn more than 50K have at least high school education? (education - 
+#     Bachelors, Prof-school, Assoc-acdm, Assoc-voc, Masters or Doctorate feature)
+
+x <- select(adult, salary, Education)
+highincomepeople <- filter(x, salary ==">50K")
+highincomepeople 
+educationdegree <- c("Bachelors", "Prof-school", "Assoc-acdm", "Assoc-voc", "Masters or Doctorate")
+educationdegree
+result <- str_detect(highincomepeople$Education, paste(educationdegree, collapse = '|'))
+result
+result_df <- data.frame(result)
+head(result_df)
+sum(result_df$result == "TRUE")
+sum(result_df$result == "FALSE")
+
```

Comment:
I don't understand what you wanted to show, 
but the answer should be smth like "no, there are people with less than higher education who earn more than 50k" and it has to be programmed correctly

-5 points